### PR TITLE
LPS-168788 and LPS-169088 Add support to the id in the fragment's drop zone tags

### PR DESCRIPTION
**Pull request by lfbesada**
https://issues.liferay.com/browse/LPS-168788
https://issues.liferay.com/browse/LPS-169088

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:stop

**Comment by lfbesada** [comment]
ci:test:sf

File: modules/apps/fragment/fragment-entry-processor/fragment-entry-processor-drop-zone/src/main/java/com/liferay/fragment/entry/processor/drop/zone/DropZoneFragmentEntryProcessor.java [hunk 1]
```diff
@@ -119,12 +120,25 @@ public String processFragmentEntryLinkHTML(
 				fragmentEntryProcessorContext.getMode(),
 				FragmentEntryLinkConstants.EDIT)) {
 
-			for (int i = 0;
-				 (i < dropZoneItemIds.size()) && (i < elements.size()); i++) {
+			boolean existIds = true;
```
Comment (ruben-pulido) [inline modules/apps/fragment/fragment-entry-processor/fragment-entry-processor-drop-zone/src/main/java/com/liferay/fragment/entry/processor/drop/zone/DropZoneFragmentEntryProcessor.java:126]: I would rename it to idsAvailable or idsExist

File: modules/apps/fragment/fragment-entry-processor/fragment-entry-processor-drop-zone/src/main/java/com/liferay/fragment/entry/processor/drop/zone/DropZoneFragmentEntryProcessor.java [hunk 2]
```diff
@@ -119,12 +121,58 @@ public String processFragmentEntryLinkHTML(
...
+				}
+			}
+			else {
+				for (int i = 0; i < elements.size(); i++) {
+					Element element = elements.get(i);
+
+					String id = element.id();
+
+					for (String itemId : dropZoneItemIds) {
+						LayoutStructureItem childrenLayoutStructureItem =
```
[the remark this answers is not in the document]
Comment (lfbesada) [reply modules/apps/fragment/fragment-entry-processor/fragment-entry-processor-drop-zone/src/main/java/com/liferay/fragment/entry/processor/drop/zone/DropZoneFragmentEntryProcessor.java]: done! thanks

**Comment by ruben-pulido** [comment]
ci:forward

**Comment by lfbesada** [comment]
ci:forward

**Comment by lfbesada** [comment]
ci:forward

**Comment by lfbesada** [comment]
This PR will conflict when forwarding to Brian's repo. I leave it On Hold until the master syncs to fix the conflicts

**Comment by lfbesada** [comment]
ci:forward

**Comment by lfbesada** [comment]
ci:forward

**Comment by lfbesada** [comment]
ci:reevaluate:1377401_6521

**Comment by ozysouza** [comment]
I will start reviewing once CI post the results :)

**Comment by ozysouza** [comment]
The failures are unrelated to this PR. Please see https://github.com/brianchandotcom/liferay-portal/pull/126787

**Pull request closed without merging**